<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tablaResultados">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>671</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <row>
     <property name="text">
      <string>ETA</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>W0</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>WF</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Iteración 1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Iteración 2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Iteración 3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Iteración 4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Iteración 5</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>_</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>_</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>_</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>_</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>_</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>_</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>_</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>_</string>
     </property>
    </item>
    <item row="1" column="3">
     <property name="text">
      <string>_</string>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string>_</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>_</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>_</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>_</string>
     </property>
    </item>
    <item row="2" column="3">
     <property name="text">
      <string>_</string>
     </property>
    </item>
    <item row="2" column="4">
     <property name="text">
      <string>_</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="entradaBias">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1, 1, 1, 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="entradaX1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0, 0, 1, 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="entradaX2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0, 1, 0, 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="entradaY">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0, 0, 0, 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="entradaETA">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>570</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RANDOM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="botonDef">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ejecutar con Atributos Predefinidos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonMod">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Modificar Atributos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonEjecutar">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>565</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ejecutar</string>
    </property>
   </widget>
   <widget class="QLabel" name="textoDef">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>290</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ejecutar el programa con valores predefinidos y aleatorios, solo modificar W0.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textoMod">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>390</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Modificar X, Y, Bias, las cinco ETAS y W0. Ejecutar luego el programa.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="botonOtraVez">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>450</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Volver a Empezar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entradaW0">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>570</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RANDOM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textoBias">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bias</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textoX1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textoX2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textoY">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>110</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textoETA">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>530</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ETA 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textoW0">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>530</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>W0 (Peso Inicial)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Perceptron</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textoUmbral">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>470</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Umbral</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="entradaUmbral">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>510</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RANDOM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tablaSecreta">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>671</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <row>
     <property name="text">
      <string>Iter</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>datos 1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>datos 2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>datos 3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>datos 4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>datos 5</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>_</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>_</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>_</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>_</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>_</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
